types/node-cron/index.d.ts

```diff
@@ -4,7 +4,11 @@
 //                 burtek <https://github.com/burtek>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function schedule(cronExpression: string, func: () => void, options?: ScheduleOptions): ScheduledTask;
+export function schedule(
+    cronExpression: string,
+    func: () => void | Promise<void>,
```

Comment:
Doesn't `() => void` cover the `async` case already? According to [Typescript's Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#callback-types) `void` return value should be used for callbacks whose value will be ignored. I don't really see any advantage of adding `Promise` to the signature here...

Reply:
You know what, you're totally right! lol

Closing this PR. Thanks

Reply:
In fact, even if Promises weren't supported explicitly, they would still workas usual ;)